dataactbroker/README.md

```diff
@@ -773,6 +773,153 @@ Example output if there are no files available:
 }
 ```
 
+# File Generation Routes
+
+## Generate Files
+**Route:** `/generate_file`
+
+**Method:** `POST`
+
+This route sends a request to the backend to utilize the relevant external APIs and generate the relevant file for the metadata that is submitted.
+
+**Deprecation Notice:** This route replaces `/generate_d1_file` and `/generate_d2_file`.
```

Comment:
This documentation was taken from Kevin's definition of the routes.  That said, it looks like this left out the /v1/ preceding these routes, so I've added that to the doc